src/lib/SCEditor.js

```diff
@@ -404,16 +404,30 @@ define(function (require) {
 
 		/**
 		 * Creates the editor iframe and textarea
+		 * @param {boolean} startInSourceMode Force loading the editor in this
+		 *																	mode
 		 * @private
 		 */
-		initEditor = function () {
+		initEditor = function (startInSourceMode) {
 			var doc, tabIndex;
 
-			$sourceEditor  = $('<textarea></textarea>').hide();
+			$sourceEditor  = $('<textarea></textarea>');
 			$wysiwygEditor = $(
 				'<iframe frameborder="0" allowfullscreen="true"></iframe>'
 			);
 
+			/* This needs to be done right after they are created because,
```

Comment:
@samclarke That doesn't work. If I do that, it will do BBCode to HTML (BBCode is an example there) transformation followed by HTML to BBCode transformation.
I don't want that to happen. I want it to actually start in source code mode and that source to be the exact content that came from the server